great-idea/css/services.css

```
@@ -0,0 +1,195 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+	width: 800px;
+	margin: auto;
+}
+
+/* Header Styles */
+
+header {
+	margin-top: 20px;
+	display: flex;
+	justify-content: space-evenly;
+	padding: 20px 0px 20px 0px;
+}
+
+header nav {
+	width: 100%;
+	display: flex;
+	justify-content: space-evenly;
+	align-self: center;
+	
+}
+
+header nav a {
+	font-size: 90%;
+	text-decoration: none;
+	color: black;
+	opacity: .7;
+	padding-right: 25px;
+}
+
+/* Banner Styles */
+
+.banner {
+	border-bottom: 2px solid black;
+}
+
+.banner img{
+	width: 100%;
+	height: auto;
+}
+
+.banner .banner-text {
+	margin: 25px 0px 25px 0px;
+	
+}
+
+.banner .banner-text h2 {
+	font-size: 25Fpx;
+}
+
+/* MAIN CONTENT STYLES */
+
+.main-content {
+	margin: 25px 0px 25px 0px;
+	height: 700px;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	align-content: space-around;
+}
+
+.main-content div {
+	border: inherit;
+	width: 45%;
+	height: 30%;
+	padding: 15px;
+	background-color:#d9d9d9;
+	border: 2px solid black;
+
+}
+
+.main-content div a {
+	display: inline-block;
```

```suggestion
	display: flex;
``` 
FLEXBOX PROJECT! You may have to refactor things if you were to change from inline-block to flex but that was the goal of this project.